Update access-provider.md (#977)
MetaMask/metamask-docs: wallet/how-to/access-provider.md

Use the [`@metamask/providers`](https://github.com/MetaMask/providers) module to access a user's
MetaMask provider from other sources, such as external extension providers and inpage providers.

> The accounts provided by this provider is the user's MetaMask account.

Reviewer: ```suggestion
The accounts provided by this provider are the user's MetaMask accounts.
```
